src/cljs/athens/views/blocks/core.cljs

```diff
@@ -143,35 +143,126 @@
       (swap! state assoc :drag-target target))))
 
 
+(defn drop-bullet
+  "
+  Terminology :
+    - DnD               : This is short for Dragged and dropped.
+    - Zero level blocks : Refers to top level blocks in a page.
+    - source-uid        : The block which is being dropped.
+    - target-uid        : The block on which source is being dropped.
+    - drag-target       : Where is the block being dragged see `types of events` section below.
```

Comment:
Bad explanation, should be 

- drag-target :  Represents where the block is being dragged. It can be `:child` meaning dragged as a child, `:above` meaning the source block is dropped above the target block, `:below` meaning the source block is dropped below the target block.